cmd/generic-handlers.go

```diff
@@ -296,7 +296,7 @@ const (
 // Check if the incoming path has bad path components,
 // such as ".." and "."
 func hasBadPathComponent(path string) bool {
-	path = strings.TrimSpace(path)
+	path = filepath.ToSlash(strings.TrimSpace(path)) // For windows '\' must be converted to '/'
```

Comment:
cherry-pick is not done properly, this code won't compile, missing import of "filepath" package